api/allocations.go

```diff
@@ -148,9 +148,6 @@ func (a *Allocations) GC(alloc *Allocation, q *QueryOptions) error {
 // Note: for cluster topologies where API consumers don't have network access to
 // Nomad clients, set api.ClientConnTimeout to a small value (ex 1ms) to avoid
 // long pauses on this API call.
-//
-// BREAKING: This method will have the following signature in 1.6.0
-// func (a *Allocations) Restart(allocID string, taskName string, allTasks bool, w *WriteOptions) (*WriteMeta, error) {
```

Comment:
I can't believe how many hints we were given that this would happen. I know I even hit it in the e2e cluster at least once but couldn't reproduce it and just assumed it was some terrible thing I had done to my e2e cluster.

Reply:
Chesterton's Fence!

Reply:
I had never heard of that before and love it!